frontend/server/src/Controllers/Problem.php

```diff
@@ -3998,32 +4036,66 @@ public static function getProblemNewForSmarty(
                         'SELECTED_TAGS' => strval($r['selected_tags']),
                         'STATUS_ERROR' => $statusError,
                         'IS_UPDATE' => false,
+                        'payload' => array_merge(
+                            [
+                                'timeLimit' => strval($r['time_limit']),
+                                'validatorTimeLimit' => strval(
+                                    $r['validator_time_limit']
+                                ),
+                                'overallWallTimeLimit' => strval(
+                                    $r['overall_wall_time_limit']
+                                ),
+                                'extraWallTime' => strval(
+                                    $r['extra_wall_time']
+                                ),
+                                'outputLimit' => strval($r['output_limit']),
+                                'inputLimit' => strval($r['input_limit']),
+                                'memoryLimit' =>  strval($r['memory_limit']),
+                                'languages' => strval($r['languages']),
+                                'validator' => strval($r['validator']),
+                            ],
+                            self::getCommonPayloadForSmarty()
+                        ),
                     ],
                     'template' => 'problem.new.tpl',
                 ];
             }
         }
+        $sortedLanguages = \OmegaUp\Controllers\Run::DEFAULT_LANGUAGES;
+        sort($sortedLanguages);
+
         return [
             'smartyProperties' => [
                 'TITLE' => '',
                 'ALIAS' => '',
                 'VALIDATOR' => \OmegaUp\ProblemParams::VALIDATOR_TOKEN,
-                'TIME_LIMIT' => '1000',
-                'VALIDATOR_TIME_LIMIT' => '1000',
-                'OVERALL_WALL_TIME_LIMIT' => '60000',
-                'EXTRA_WALL_TIME' => '0',
-                'OUTPUT_LIMIT' => '10240',
-                'INPUT_LIMIT' => '10240',
-                'MEMORY_LIMIT' => '32768',
                 'EMAIL_CLARIFICATIONS' => '0',
                 'SOURCE' => '',
                 'VISIBILITY' => '0',
+                'STATUS_ERROR' => '',
                 'LANGUAGES' => join(
                     ',',
                     \OmegaUp\Controllers\Run::DEFAULT_LANGUAGES
                 ),
                 'SELECTED_TAGS' => '',
                 'IS_UPDATE' => false,
+                'payload' => array_merge(
+                    [
+                    'timeLimit' => 1000,
```

Comment:
nit:

```suggestion
                        'timeLimit' => 1000,
```

así es claro que todas estas cosas pertenecen a un arreglo.